docs/data-sources/certificate.md

```diff
@@ -1,13 +1,12 @@
 ---
-layout: "vcfa"
 page_title: "VMware Cloud Foundation Automation: vcfa_certificate"
-sidebar_current: "docs-vcfa-data-source-certificate"
+subcategory: ""
 description: |-
   Provides a data source to read a Certificate in VMware Cloud Foundation Automation. This can be used to read the certificates that
   VCF Automation provides to others. They can be used when creating services that must be secured.
 ---
 
-# vcfa\_certificate
+# Data Source: vcfa_certificate
```

Comment:
Sorry, but I don't see this as improvement. Out of four largest Haschicorp providers only AWS uses this:
https://registry.terraform.io/namespaces/hashicorp

But its left menu hierarchy is vastly different.

In our case, it's clear whether you're reading resource or data source, because you select the resource/data source folder on the left meniu.
The issue is that having this in the title is actually making harder to read/scan what resource are you looking at.